templates/rst_display.html

```diff
@@ -6,8 +6,24 @@
 
 {% block title %}{{ rst.title }}{% endblock %}
 
+{% block og_type %}{{ 'article' if ctx.pub_date else 'website' }}{% endblock %}
 {% block og_url %}{{ get_full_canonical_url(ctx.builder, slug=ctx.slug) }}{% endblock %}
 {% block og_summary %}{{ ctx.render_summary()|striptags }}{% endblock %}
+{% block twitter_type %}{{ 'summary_large_image' if ctx.config.image and ctx.config.image[0].src else 'summary' }}{% endblock %}
+{% block extra_og %}
+  {% if ctx.pub_date %}
```

Comment:
I'm sort of using `ctx.pub_date` as the signifier for a blog post. Is there a better way?